test/cartesian/getEquidistantTicks.spec.ts

```diff
@@ -36,3 +32,44 @@ describe('getEquidistantTicks', () => {
     expect(result).toEqual(resultingTicks.map(index => ticks[index]));
   });
 });
+
+describe('getEquidistantPreserveEndTicks', () => {
+  const getTickSize = (tick: CartesianTickItem) => {
+    return tick.value;
+  };
+
+  it('should return empty array if no ticks are passed', () => {
+    const result = getEquidistantPreserveEndTicks(1, { start: 0, end: 100 }, getTickSize, [], 0);
+    expect(result).toEqual([]);
+  });
+
+  test.each([
```

Comment:
The test cases look like testing `identity` function. What's the difference between `getEquidistantPreserveEndTicks` and identity?